### PR TITLE
Monkeypox demographics schema updates

### LINKED ISSUE #6275: Add demographic fields and remove patient state requirement for monkeypox
## Problem statement

Need to add 4 demographic fields to dcipher monkeypox schema:

1. subject_age
2. subject_race
3. subject_ethinicity
4. subject_sex

And make patient_state optional for monkeypox senders. We do not want to reject the message if the patient state is not present.

### PR DESCRIPTION
**Pull request by jbiskie**
This PR adds race, ethnicity, sex, and age to the dcipher schema and makes patient_state non-required.

Test Steps:
1. Process a sample monkeypox HL7 message, confirm subject_race, subject_ethnicity, subject_sex, and subject_age columns are present in the dcipher output.   ** Note: subject_age is not yet populating
3. Process a sample monkeypox HL7 message without pid-11.4, confirm message does not fail.

## Changes
- added subject_race, subject_ethnicity, subject_sex, and subject_age to dcipher-monkeypox.schema
- changed patient_state cardinality to ZERO_OR_ONE in monkeypox.schema

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #6275

### CODE REVIEW

File: prime-router/metadata/schemas/monkeypox.schema [hunk 1]
```diff
@@ -572,7 +569,7 @@ elements:
     pii: false
     table: fips-county
     tableColumn: State
-    cardinality: ONE
+    cardinality: ZERO_OR_ONE
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/monkeypox.schema:572]: This is fine. We can probably remove the table components as well and just make it pure text if we want.

**Review by MauriceReeves-usds — APPROVED** [review]
One comment, but looks good to me.

File: prime-router/metadata/schemas/monkeypox.schema (hunk 1 above)
Comment (jbiskie) [reply prime-router/metadata/schemas/monkeypox.schema:572]: Yeah, I debated on that too.

**Review by jimduff-usds — APPROVED** [review]
lgtm

**Pull request merged**